src/views/v2/rsvp/ari/form/guest-template.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * This template renders the RSVP AR form guest template.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form/guest-template.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+<script
+	class="tribe-tickets__rsvp-ar-form-guest--template"
```

Comment:
a technicality, but the class should be something like `tribe-tickets__rsvp-ar-form-guest-template`, since the `<script>` tag isn't a modification of the `tribe-tickets__rsvp-ar-form-guest` element.